.github/workflows/abi-check.yml

```diff
@@ -0,0 +1,22 @@
+name: ABI Check
```

Comment:
I wonder if instead of a CI check we could run this automatically with something like https://github.com/hkdobrev/run-if-changed. Not a blocker though :)

Reply:
Yeah.. We can use pre-commit hooks that we use in other repos, but as I expect changes to this repo to be minimal I didn't add it 😅 

I'll consider it in a separate PR if I go for it, to also make sure the contract compiles. (I'll create an issue so we don't forget it)